data/migrations/20190911083741_[cars.js].js

```diff
@@ -0,0 +1,24 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", tbl => {
+    tbl.increments();
+    // VIN
+    tbl
+      .string("VIN", 128)
+      .unique()
+      .notNullable();
+    // MAKE
+    tbl.string("MAKE", 128).notNullable();
+    // MODEL
+    tbl.string("MODEL", 128).notNullable();
+    // MILEAGE
+    tbl.integer("MILEAGE").notNullable();
+    // transmission type
+    tbl.string("TRANSMISSION TYPE", 128);
+    //status of the title (clean, salvage, etc.)
+    tbl.string("TITLE STATUS", 128);
+  });
+};
+
+exports.down = function(knex) {
+  return knex.schema.dropTableIfExists("cars");
+};
```

Comment:
nice table setup, i would say change some of the char limits, if you want it to be really specific, so like millage can have a character limit of maybe 30, since a car would most likely be destroyed before reaching that many miles, and not to many car manufactures or car names are too long so those can be limited to 40-50 as well